src/identity/components/GlobalHeader/GlobalHeader.tsx

```diff
@@ -77,33 +77,42 @@ export const GlobalHeader: FC = () => {
     }
   }, [orgsList])
 
```

Comment:
These conditionals ensure that we don't render a non-functional globalheader to the user in the case where quartz endpoints fail. Rather than having an unpopulated dropdown or profile avatar, the global header will simply not render if quartz doesn't load. 

The figma doesn't specify one way or another, but my preference is not to have a toast notification for the user in this case. If quartz goes down, the user won't be able to change accounts or orgs, but they may well be able to use other functionality like data explorer until quartz comes back online, and won't want to be pestered with notifications about functionality that is only used once in a while.